htdocs/core/class/html.form.class.php

```diff
@@ -6460,7 +6460,7 @@ public function load_cache_vatrates($country_code)
 	 *                         Si vendeur non assujeti a TVA, TVA par defaut=0. Fin de regle.
 	 *                         Si le (pays vendeur = pays acheteur) alors la TVA par defaut=TVA du produit vendu. Fin de regle.
 	 *                         Si (vendeur et acheteur dans Communaute europeenne) et bien vendu = moyen de transports neuf (auto, bateau, avion), TVA par defaut=0 (La TVA doit etre paye par l'acheteur au centre d'impots de son pays et non au vendeur). Fin de regle.
-	 *                         Si vendeur et acheteur dans Communauté européenne et acheteur= particulier alors TVA par défaut=TVA du produit vendu. Fin de règle.
+	 *                         Si vendeur et acheteur dans Communauté européenne et acheteur= particulier alors TVA par défaut=TVA du pays acheteur. Fin de règle.
```

Comment:
I think the term "du produit vendu" was the good one so if we sell wine in a restaurant it may be vat 10% and if meet, it may be 20%).  We use the vat of selling country only if product is unknow.

Reply:
I understand what you mean but, there is a problèm with the main reason why this rule was introduced.
If you sell meat to an EEC country, you have to apply the VAT of this country. 
This calls into question the structure of VAT rates because there is a lack of a notion of “VAT class” in order to be able to transpose VAT between different countries. ex: VAT 20% (FR) <=> VAT 19%(DE)